src/utils/images.ts

```diff
@@ -0,0 +1,4 @@
+const ALLOWED_HOSTS = ['https://media.guim.co.uk/', 'https://i.guim.co.uk/'];
+
+export const isImageUrlAllowed = (imageUrl: string): boolean =>
+    ALLOWED_HOSTS.some((host) => imageUrl.startsWith(host));
```

Comment:
I noticed that `String.prototype.startsWith` isn't supported natively on IE11, but we polyfill it (confirmed on BrowserStack).